src/components/Header.js

```diff
@@ -55,10 +51,7 @@ const Header = () => {
                 isOpen={isModalOpen}
```

Comment:
Rename `ModalComponent`.  Because I don't understand what content this component has.